boot_default/config.txt

```diff
@@ -57,6 +57,11 @@ start_x=0
 # Note: RaspberryPI model B+ only
 max_usb_current=0
 
+
+# Note: kano_settings.boot_config.check_corrupt() checks for presence of
+#       "dtparam" to determine if this file is corrupt, don't remove without
+#       updating that function
```

Comment:
At a minimum I was thinking we'd have tests to verify that what the function was looking is in this file. 
But I guess this is the next minimum...